node_pp_name: "person_pose"

subscribers:

  person_object_flag:
    topic: /affordances/flag
    queue_size: 1

publishers:

  status_person_pose:
    topic: /affordances/status_person_pose
    queue_size: 1
